src/routers/people/index.ts

```diff
@@ -1,166 +1,133 @@
-import createRouter from '@/utilities/functions/createRouter';

```

Comment:
why did this file change? looks the same?

Reply:
line endings :)